src/main/kotlin/com/hedvig/rapio/externalservices/memberService/MemberService.kt

```diff
@@ -16,12 +29,94 @@ class MemberService(
                 return response.body!!
             }
         } catch (ex: Exception) {
-            logger.error("Error from member service when checking member status for ${IsMemberRequest(memberId, ssn, email)}.", ex)
+            logger.error(
+                "Error from member service when checking member status for ${
+                    IsMemberRequest(
+                        memberId,
+                        ssn,
+                        email
+                    )
+                }.", ex
+            )
         }
         return false
     }
 
+    fun createMemberWithTrialInsurance(
+        language: String,
+        countryCode: CountryCode,
+        partner: Partner,
+        fromDate: LocalDate,
+        newMemberInfo: NewMemberInfo
+    ): Long {
+        val memberId = memberServiceClient.createMember(
+            CreateMemberRequest(language, partner.toString())
+        ).bodyOrNull()?.memberId ?: throw internalServerError()
```

Comment:
Adding a `toString` here would save us quite a lot of random `toString`s later
```suggestion
        val memberId = memberServiceClient.createMember(
            CreateMemberRequest(language, partner.toString())
        ).bodyOrNull()?.memberId.toString() ?: throw internalServerError()
```